src/js/tabs/osd.js

```diff
@@ -96,7 +96,7 @@ FONT.constants = {
 * Each line is composed of 8 asci 1 or 0, representing 1 bit each for a total of 1 byte per line
 */
 FONT.parseMCMFontFile = function (data) {
-    var data = data.split("\n");
+    var data = data.trim().split("\n");
```

Comment:
I got advised on the slack that I should just trim() the file to remove the newline. I still think there is merit to only accepting full characters.